### ISSUE TITLE
Grab/integrate metadata related documentation from IndustryPortal

### ISSUE THREAD

**Issue by jonquet** [OntoPortal/administration#40]
Discovered this doc to be integrtaed/merged in our general doc
https://industryportal.github.io/submission-guide

**Comment by arsarkar** [comment]
Yes, this document is 80% complete but if it is merged into the general doc we can redirect and work from there. 


From: "Clement Jonquet" ***@***.***> 
To: "ontoportal/documentation" ***@***.***> 
Cc: "Arkopaul Sarkar" ***@***.***>, "Assign" ***@***.***> 
Sent: Wednesday, 8 November, 2023 09:38:52 
Subject: [ontoportal/documentation] Grab/integrate metadata related documentation from IndustryPortal (Issue #40) 





Discovered this doc to be integrtaed/merged in our general doc 
[ https://industryportal.github.io/submission-guide | https://industryportal.github.io/submission-guide ]
